data/json/furniture_and_terrain/furniture-appliances.json

```diff
@@ -546,7 +546,7 @@
         { "item": "sheet_metal", "count": [ 2, 4 ] },
         { "item": "pipe", "count": [ 1, 4 ] },
         { "item": "cable", "charges": [ 1, 15 ] },
-        { "item": "motor", "count": 1 },
+        { "item": "motor_tiny", "count": 1 },
```

Comment:
So if I'm reading this correctly the paper shredder has a 3lb motor?  I'll come back to this later today (hopefully) to keep reviewing.

Reply:
Yeah, it WAS a 100 lb / 46 kg motor.  That one was way overkill.

![image](https://user-images.githubusercontent.com/34361592/230621938-c1620fba-4ac4-4fa7-b544-89f705939737.png)

Reply:
I googled what paper shredder for home offices weigh and it's 8-9 lbs and since it's basically motor, blades and trash can 1/3rd being motor makes sense.